### PR TITLE
[PeerDAS] Move custody subnet count into helper function

### PR DESCRIPTION
**Pull request by 0x00101010**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**
Move custody subnet count into helper function

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

File: beacon-chain/core/peerdas/helpers.go [hunk 1]
```diff
@@ -300,3 +302,13 @@ func VerifyDataColumnSidecarKZGProofs(sc *ethpb.DataColumnSidecar) (bool, error)
 	}
 	return cKzg4844.VerifyCellKZGProofBatch(ckzgComms, rowIdx, colIdx, cells, proofs)
 }
+
+// CustodySubnetCount returns the number of subnets the node should participate in for custody.
+func CustodySubnetCount() uint64 {
+	// TODO: add ability to customize custody requirement.
```
Comment (nalepae) [inline beacon-chain/core/peerdas/helpers.go:308]: At the moment, it is not sure we'll allow that.
Right now, it is possible to:
- Either custody the minimum amount of columns, or
- Either custody all the columns


With validator custody, the number of custody columns will depend on the ETH amount managed by attached validators.

File: beacon-chain/p2p/subnets.go [hunk 2]
```diff
@@ -220,7 +220,7 @@ func initializePersistentColumnSubnets(id enode.ID) error {
 	if ok && expTime.After(time.Now()) {
 		return nil
 	}
-	subsMap, err := peerdas.CustodyColumnSubnets(id, params.BeaconConfig().CustodyRequirement)
+	subsMap, err := peerdas.CustodyColumnSubnets(id, peerdas.CustodySubnetCount())
```
Comment (nalepae) [inline beacon-chain/p2p/subnets.go:223]: Nice catch!

File: beacon-chain/sync/initial-sync/blocks_fetcher.go [hunk 3]
```diff
@@ -575,8 +576,9 @@ func populateBlockWithColumns(bw blocks2.BlockWithROBlobs, columns []blocks.RODa
 		return bw, errDidntPopulate
 	}
 	colsPersub := params.BeaconConfig().NumberOfColumns / params.BeaconConfig().DataColumnSidecarSubnetCount
-	if len(columns) != int(params.BeaconConfig().CustodyRequirement*colsPersub) {
-		return bw, errors.Errorf("unequal custodied columns provided, got %d instead of %d", len(columns), int(params.BeaconConfig().CustodyRequirement))
+	subnetCount := peerdas.CustodySubnetCount()
```
Comment (nalepae) [inline beacon-chain/sync/initial-sync/blocks_fetcher.go:579]: @nisdas could you please checks changes in this file are OK?

Comment (nisdas) [reply beacon-chain/sync/initial-sync/blocks_fetcher.go:579]: Looks good to me

**Pull request merged**